### DISCUSSION TITLE
Consiglio su Raspberry

### DISCUSSION THREAD

**Discussion by millenniumy2k** [indomus/forum#1236] [Hardware & Firmware]
Dopo aver fatto le mie mille prove su Windows 10 e Docker, volevo portare tutto in produzione, pensavo quindi, dovendo stare accesso h24, a un Raspberry pi4.
Va bene la versione da 4gb (i prezzi per la 8gb non sono affrontabili ora...)?

**Comment by snakuzzo** [comment]
Ma perché aprire un topic per fare questa domanda ?
Non bastava chiedere al volo sulla chat Telegram ?

**Reply by millenniumy2k** [reply]
...forse perchè non uso telegram... :-P

**Reply by snakuzzo** [reply]
Ma capirai che aprire topic così per consigli del genere così banali e diffusi, mi pare una cavolata.
Tra le altre cose ti basta fare qualche ricerca qui sul forum per trovare altri topic simili.
Cercare prima di chiedere...è la prima regola di qualsiasi forum su questo pianeta

**Reply by millenniumy2k** [reply]
Banale per te forse, ci sono altre discussioni dove si chiedono consigli... linkami una risposta alla mia domanda, io non ne ho trovate.

**Reply by snakuzzo** [reply]
Vedo che ti hanno risposto. A posto così.

**Comment by KC-inDomus** [comment]
> Dopo aver fatto le mie mille prove su Windows 10 e Docker, volevo portare tutto in produzione, pensavo quindi, dovendo stare accesso h24, a un Raspberry pi4.
> 
> Va bene la versione da 4gb (i prezzi per la 8gb non sono affrontabili ora...)?

la versione 4gb del raspi 4 è strasufficiente, già la versione 2gb andava bene ma 4 è certamente più cautelativa rispetto all'allocazione in memoria di più processi. 
quanto ai prezzi... è tutto (ma proprio tutto) ai massimi, purtroppo colpa del covid. un anno fa costava almeno il 25% di meno, ma c'è poco da fare.

**Reply by millenniumy2k** [reply]
grazie per la tua risposta.

**Reply by KC-inDomus** [reply]
figurati 👍
